src/groupByKey.ts

```diff
@@ -2,6 +2,20 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+type Item = object;
```

Comment:
This extra Type is unnecessary

Reply:
There are several ways to deal with that type. I still have doubts about which is the best...